### DISCUSSION TITLE
89900a3a-2e3d-4d72-b935-0949bd1cd8ed

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9586] [Rules Discussions]
# 89900a3a-2e3d-4d72-b935-0949bd1cd8ed

Do you 'zz' old files rather than deleting them? Learn how to preserve your outdated .NET application releases and Microsoft Word proposals by prefixing filenames with 'zz'.

https://www.ssw.com.au/rules/zz-files/

**Comment by sylhuang** [comment]
The YouTube part of this rule is really useful when someone opens an old version via using an outdated link e.g on a website or sent to someone. 

When the zzRule is applied, viewers see the old video (no broken link), and they observe that they are seeing an outdated version (via “zz” in the title), and are able to navigate to the current version (via the new link in description).

**Comment by jernejk** [comment]
It's very useful to link new video URL on the old video.

**Comment by JackDevAU** [comment]
Good rule very useful!

Tip: append the date too (e.g. zz-2026-04-30-filename) so you know when to clean up

**Comment by designbyalex** [comment]
Useful for people to navigate between versions of YouTube videos... and avoid watching an outdated version.

**Comment by 0xharkirat** [comment]
zz means "nah fam, scroll past, I'm last season"... 

This rule has kept me sane, when looking at many versions of same file.

**Comment by Lewkans** [comment]
Useful to ensure that someone won't visit a broken link, only an outdated link with the link to the latest version.

**Comment by kulesy** [comment]
Great to read up on this again, very important so we leave an audit trail.
